packages/react-components/react-table/src/hooks/useTableCompositeNavigation.ts

```diff
@@ -9,7 +9,7 @@ import {
   useFocusFinders,
 } from '@fluentui/react-tabster';
 import { isHTMLElement } from '@fluentui/react-utilities';
-import { TabsterTypes, dispatchGroupperMoveFocusEvent, dispatchMoverMoveFocusEvent } from '@fluentui/react-tabster';
+import { TabsterTypes, TabsterEvents } from '@fluentui/react-tabster';
```

Comment:
@mshoho going forward I'd suggest to stop using classes with namespace API's for 2 very important reasons:
- bundlers have issues with classes/namespaces tree shaking 
- mixing types with runtime tokens is an antipattern and might lead to future issues like happened to `TabsterTypes`